requirements-dev.txt

```diff
@@ -2,3 +2,4 @@ pytest==2.8.5
 pytest-cov==2.2.0
 pytest-pep8==1.0.6
 typing==3.6.1
+-e git+https://github.com/python/mypy@005fd6b357b995f13e3d90293cabf3c8c8c2a952#egg=mypy
```

Comment:
I assume this is needed for travis to run correctly. Will this dependency be installed if someone uses pip to install the package for python 2? If so I'm not sure that is a good idea as mypy is not a python 2 package.